README.md

```diff
@@ -1,2 +1,17 @@
 # MusicGenreRecognition
 Machine learning project on music genre recognition
+
+# Driver
+Driver will iterate through all songs. For each song, it will call extract() on each of our classes. It will take the returned features and concatenate them to a list of all features for that song. It will add this list as a row to the .arff file
+- The arguments for extract will be (full song, song section, sampling rate)
```

Comment:
The ordering of parameters isn't consistent with what I put in the driver.  I had thought we decided that the sample would be the first parameter, and the full song would be the second.  If that's not the case then I can update the driver.

Reply:
Oh my bad, I remembered wrong then. I'll change it.